subsys/bluetooth/mesh/rpr_srv.c

```diff
@@ -200,7 +200,8 @@ static void scan_report_schedule(void)
 		return;
 	}
 
-	delay = (sys_rand32_get() % 480) + 20;
+	(void)bt_rand(&delay, sizeof(uint32_t));
```

Comment:
Is it OK to silently fail this?

Reply:
This is a recommendation to use random delay during scan reporting, to reduce probability of collisions. If random generator doesn't work, I think it is still better to report scanning result with not random delay rather than silently to stop working.

Reply:
@alxelax if it fails, it means that `delay` will not be initialized at all. Although that's a kind of "randomness" as well, it's not explicit and will likely cause static analyzer warnings. It would therefore probably be better to at least initialize it to `0` or something like that.